charts/seed-bootstrap/charts/fluent-bit/templates/_fluent-bit-config.tpl

```diff
@@ -91,24 +91,34 @@
         Match kubernetes.*
         Url http://loki.garden.svc:3100/loki/api/v1/push
         LogLevel info
-        BatchWait 30
-        # (30sec)
+        BatchWait 40
+        # (40sec)
```

Comment:
What is the purpose of this comment?

Reply:
to show that time is measured in seconds.

Reply:
Why it is not on the same line with BatchWait ?

Reply:
copy->paste from the original plugin. If you want I will put it on the same line.

Reply:
Either remove it or put it on the same line.

Reply:
ok

Reply:
putting comment in the same line resolve in adding it to the particular value.
Could I put on top?

Reply:
Remove them.